seminars/talk.py

```diff
@@ -123,6 +124,10 @@ def editable_end_time(self):
         """
         return self._editable_time(self.end_time)
 
+    @property
+    def tz(self):
+        return pytz.timezone(self.timezone)
```

Comment:
We might want to protect against `timezone` being invalid (e.g. `None`), though I think we don't allow that.

Reply:
We don't do this in seminar,py, so I don't think we need to (but if we do we should do it in both places.